medschool/src/parse.rs

```diff
@@ -0,0 +1,286 @@
+//! All parsing of `syn::File`` and `syn::Item`` into relevant types and fields related functions.
+
+use std::collections::{BTreeSet, HashMap, HashSet};
+
+use syn::{Attribute, Expr, Ident, Meta, Type, TypePath};
+
+use crate::{
+    types::{PartialField, PartialType},
+    DocsError,
+};
+
+#[tracing::instrument(level = "trace", ret)]
+pub fn docs_from_attributes(attributes: Vec<Attribute>) -> Vec<String> {
```

Comment:
```suggestion
pub fn docs_from_attributes(attributes: Vec<Attribute>) -> Option<Vec<String>> {
```

wdyt? Then in every function that uses this, you can just do `let docs = docs_from_attributes(attrs)?;`

Reply:
what is the use of it tho?